docling_core/types/doc/document.py

```diff
@@ -1775,6 +1851,7 @@ def iterate_items(
         with_groups: bool = False,
         traverse_pictures: bool = False,
         page_no: Optional[int] = None,
+        included_content_layers: List[ContentLayer] = [ContentLayer.BODY],
```

Comment:
let's make this a set, and save the default in a global var. then users can do `included_content_layers=DEFAULT + ContentLayer.BLAH`